.github/README.md

```diff
@@ -60,26 +82,39 @@ After that, you can just open the `UndertaleModTool.sln` file in Visual Studio,
 Alternatively, you can also compile via command line, like so: `dotnet publish UndertaleModTool --no-self-contained -p:PublishSingleFile=true -c release -r win-x86`
 You can adjust win-x86 to other RIDs, such as win-x64 or win-arm.
 
-### data.win file format
+# data.win File Format
+
 Interested in the file and instruction format research I've done while working on this? Check out these:
-* https://github.com/krzys-h/UndertaleModTool/wiki/Corrections-to-GameMaker-Studio-1.4-data.win-format-and-VM-bytecode,-.yydebug-format-and-debugger-instructions
-* https://github.com/krzys-h/UndertaleModTool/wiki/Changes-in-GameMaker-Studio-2
-* https://github.com/krzys-h/UndertaleModTool/wiki/Extensions,-Shaders,-Timelines-format
-* https://github.com/krzys-h/UndertaleModTool/wiki/Bytecode-version-differences
-* https://github.com/krzys-h/UndertaleModTool/wiki/YYC-games
 
-### Special thanks
+* [Corrections to GameMaker Studio 1.4 data.win format and VM bytecode, .yydebug format and debugger instructions](https://github.com/krzys-h/UndertaleModTool/wiki/Corrections-to-GameMaker-Studio-1.4-data.win-format-and-VM-bytecode,-.yydebug-format-and-debugger-instructions)
+* [Changes in GameMaker Studio 2](https://github.com/krzys-h/UndertaleModTool/wiki/Changes-in-GameMaker-Studio-2)
+* [Extensions, Shaders, Timelines format](https://github.com/krzys-h/UndertaleModTool/wiki/Extensions,-Shaders,-Timelines-format)
+* [Bytecode version differences](https://github.com/krzys-h/UndertaleModTool/wiki/Bytecode-version-differences)
+* [YYC games](https://github.com/krzys-h/UndertaleModTool/wiki/YYC-games)
+
+feel free to check out the [Wiki](https://github.com/krzys-h/UndertaleModTool/wiki) for more.
```

Comment:
*All* of the links are on the wiki, so why not just link directly to the wiki? Would clutter the readme less, and still get the point across.

Reply:
Also why does the YYC games page still exist, it's been empty for like 4 years now.